### PR TITLE
fix test-task-metadata->pod unit test

### PR DESCRIPTION
**Pull request by nsinkov**
## Changes proposed in this PR

fix test-task-metadata->pod unit test

### CODE REVIEW

File: scheduler/test/cook/test/kubernetes/api.clj [hunk 1]
```diff
@@ -79,19 +79,19 @@
         (let [workdir-volume (->> pod
                                   .getSpec
                                   .getVolumes
-                                  (filter (fn [^V1Volume v] (= "cook-workdir" (.getName v))))
+                                  (filter (fn [^V1Volume v] (= "cook-workdir-volume" (.getName v))))
                                   first)]
           (is (not (nil? (.getEmptyDir workdir-volume)))))
```
Comment (scrosby) [inline scheduler/test/cook/test/kubernetes/api.clj]: There should be an assert that the cook-workdir-volume is not nil.

File: scheduler/test/cook/test/kubernetes/api.clj [hunk 2]
```diff
@@ -79,19 +79,19 @@
         (let [workdir-volume (->> pod
```
Comment (scrosby) [inline scheduler/test/cook/test/kubernetes/api.clj]: Rename to cook-workdir-volume (for consistency)

File: scheduler/test/cook/test/kubernetes/api.clj [hunk 3]
```diff
@@ -79,19 +79,19 @@
...
+                                  (filter (fn [^V1Volume v] (= "cook-workdir-volume" (.getName v))))
                                   first)]
           (is (not (nil? (.getEmptyDir workdir-volume)))))
 
         (let [^V1Container container (-> pod .getSpec .getContainers first)]
           (is (= "required-cook-job-container" (.getName container)))
           (is (= ["/bin/sh" "-c" "foo && bar"] (.getCommand container)))
           (is (= "alpine:latest" (.getImage container)))
-          (is (= 3 (count (.getEnv container))))
+          (is (= 4 (count (.getEnv container))))
```
Comment (scrosby) [inline scheduler/test/cook/test/kubernetes/api.clj]: Assert getEnv is not nil.

File: scheduler/test/cook/test/kubernetes/api.clj [hunk 4]
```diff
@@ -79,19 +79,19 @@
...
           (is (= "/mnt/sandbox" (.getWorkingDir container)))
           (let [workdir-mount (->> container
```
Comment (scrosby) [inline scheduler/test/cook/test/kubernetes/api.clj]: Why workdir-mount here and workdir-volume above?

Comment (nsinkov) [reply scheduler/test/cook/test/kubernetes/api.clj]: those are different

**Comment by scrosby** [comment]
@nsinkov will fix the travis integration test failure in another PR. Interanl green

**Pull request merged**